src/app/components/core/north/north.component.html

```diff
@@ -3,7 +3,7 @@
     <header class="card-header">
       <div id="north-tasks" class="card-header-title">
         North Instances
-        <button *ngIf="!isAlive" (click)="getNorthTasks(true)" class="button is-small" id="refresh-check">
+        <button *ngIf="!isAlive" (click)="getNorthTasks(true)" class="button is-small" title="Reload" id="refresh-check">
```

Comment:
Add for all cards + Asset readings; note: for warning boxes of service added/installed this will be Recheck. Reload + Recheck can be title only when not in group